packages/shared/src/contexts/FeaturesContext.tsx

```diff
@@ -36,11 +47,19 @@ export const FeaturesContextProvider = ({
   children,
   flags,
 }: FeaturesContextProviderProps): ReactElement => {
+  const steps = getFeatureValue(Features.OnboardingSteps, flags)?.split?.('/');
+  const onboardingSteps = (steps as OnboardingStep[]) || [];
   const features = useMemo(
     () => ({
       flags,
-      feedFilterVersion: getFeatureValue(Features.FeedFilterVersion, flags),
-      feedFilterCardVersion: getFeatureValue(Features.FeedFilterVersion, flags),
+      onboardingSteps,
+      onboardingMinimumTopics:
+        getFeatureValue(Features.OnboardingMinimumTopics, flags) ?? 0,
```

Comment:
Is zero going to be the default, remember from back in the discussions 2 was the magic number?

Reply:
From the meeting we had about the experiments, it was mentioned to instead have 0. With that, I will send out a message in the channel for final confirmation. Thank you! 🚢

Reply:
Alright if you guys discussed it, it's fine 🙏

Reply:
Yes. Zero is control at the moment :)